.github/workflows/autofix.yml

```
@@ -64,10 +64,13 @@ jobs:
-            yarn prettier -c .
+            yarn prettier --write -c .
           else
-            echo "$changed_files" | xargs -r yarn prettier -c --ignore-unknown
+            echo "$changed_files" | xargs -r yarn prettier --write -c --ignore-unknown
           fi
 
       - name: Type Check
-        run: yarn tsc --noEmit
\ No newline at end of file
+        run: yarn tsc --noEmit
+        
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)